app/youtube/components/Home/HomeYoutube.tsx

```diff
@@ -9,47 +10,104 @@ const HomeYoutube = () => {
   const channelBanner = items[0].brandingSettings?.image.bannerExternalUrl;
   return (
     <main>
-      <header className={styles.youtube__navBarSection}>
-        <input type="text" placeholder="Search" />
-        <button type="submit">Lupa</button>
+      <header className={styles.youtube__searchbarSection}>
```

Comment:
should refactor the code in a way that we have multiple logic sections. 
Here for example we could create a new component that separates the logic of searching through YouTube